Assets/HoloToolkit/Input/Tests/Scenes/MotionControllerTest.unity

```diff
@@ -607,6 +7105,347 @@ Prefab:
       value: 
       objectReference: {fileID: 1097318689274706, guid: 869d340c69dbcef459070c1a159d38ac,
         type: 2}
+    - target: {fileID: 114710303647202208, guid: d29bc40b7f3df26479d6a0aac211c355,
+        type: 2}
+      propertyPath: RightControllerOverride
```

Comment:
Please revert this override. My intention was to show one override and one controller model in the test scene.

Reply:
Oh, I see. I'll update. I'll add some details about this(game mode vs deploy) to the README too.